dataProcessing.py

```diff
@@ -28,9 +28,10 @@ def parse_file(filename, prob_cutoff, db=''):
 def fill_data_dict(nhits, hitList, db):
     if db in ['pdb', 'pfam', 'yeast']:
         x1, x2, dx, y, pcent, name, detail = fill_data(nhits, hitList, db)
-        data = dict(x1=x1, x2=x2, dx=dx, y=y, name=name, pcent=pcent, detail=detail)
+        data = dict(x1=x1, x2=x2, xm=[beg+dif/2 for beg,dif in zip(x1,dx)], dx=dx, y=y, name=name, pcent=pcent, detail=detail)
+        data, ymax = squash_data(data)
         
-        return len(x1), data
+        return ymax, data
```

Comment:
It might be worth making `squash_data` return values in the same order as this, to reduce changes for future confusion!

Reply:
done